### PR TITLE
Show file name in the long press menu, see #559

### PR DESCRIPTION
**Pull request by AndyScherzinger**
solution as discussed and decided on in issue #559

### CODE REVIEW

**Comment by AndyScherzinger** [comment]
device-2015-09-07-120845
device-2015-09-07-120832

**Comment by AndyScherzinger** [comment]
imho ready for code review @davivel @tobiasKaminsky and most likely also for testing @masensio since it is a minimal code change :smiley:

**Comment by jancborchardt** [comment]
Looks good!

**Comment by AndyScherzinger** [comment]
@davitol @masensio looking good code wise? :smiley:

**Comment by cdamken** [comment]
I like it :smile:

**Comment by rperezb** [comment]
Thanks @AndyScherzinger !
I have just checked it a tablet v4.4.4 and everything is working fine.
If @MTRichards agrees we will take it into account for next sprint

**Comment by AndyScherzinger** [comment]
Just implemented it since you asked for it and it was a 15 minutes coding job :grin:

**Comment by AndyScherzinger** [comment]
@davivel fancy a code review? QA has already been done :smiley:

File: res/layout/file_actions.xml [hunk 1]
```diff
@@ -3,12 +3,31 @@
...
-    android:layout_margin="@dimen/standard_margin">
+    android:layout_margin="@dimen/standard_margin"
+    android:orientation="vertical">
+
+    <TextView
+        android:id="@+id/file_actions_header"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:padding="@dimen/standard_padding"
+        android:textColor="@color/black"
```
Comment (davivel) [inline res/layout/file_actions.xml]: What is the default color if not specified?

**Comment by davivel** [comment]
Code reviewed, seems fine :+1:

Pending of pruning to merge.

File: res/layout/file_actions.xml (hunk 1 above)
Comment (AndyScherzinger) [inline res/layout/file_actions.xml]: The default (don't know why, must be within app compat) is #6D6D6D:
device-2015-09-10-142459

**Comment by davivel** [comment]
OK. Will do some more QA in a different device, then this is in. Just got top priority.

**Comment by AndyScherzinger** [comment]
Wow, okay, cool! Not that I personally care to much about it, but I guess you will make @cdamken and @diggy128 very happy :grin:  :grin:  :grin:

**Comment by davivel** [comment]
So I hope :D

**Comment by davivel** [comment]
Tested! So, there it goes.

**Comment by cdamken** [comment]
> Wow, okay, cool! Not that I personally care to much about this feature, but I guess you will make @cdamken and @diggy128 very happy :grin: :grin: :grin:

@AndyScherzinger  Thanks, I'm happy because I will have happy users :+1:

**Comment by AndyScherzinger** [comment]
Glad I could help @cdamken :)

**Comment by diggy128** [comment]
@AndyScherzinger 
Cool. Thanks a lot for implementing this.
@cdamken 
Happy user here. :smiley: 

Will test as soon as the version comes out

**Pull request merged**